index.xml

```diff
@@ -14892,6 +15172,11 @@ d.type = et;
           <prod nt='ExtendedAttributeIdentList'>identifier "=" "(" IdentifierList ")"</prod>
           <prod nt='ExtendedAttributeNamedArgList'>identifier "=" identifier "(" ArgumentList ")"</prod>
 
+          <prod nt='Namespace'>"namespace" identifier "{" NamespaceMembers "}" ";"</prod>
+
+          <prod nt='NamespaceMembers'>ExtendedAttributeList NamespaceMember NamespaceMembers | ε</prod>
+          <prod nt='NamespaceMember'>ReturnType OperationRest</prod>
```

Comment:
`CSS.elementSources` is an attribute. https://github.com/w3c/csswg-drafts/issues/428